hack/goreleaser.sh

```diff
@@ -43,6 +43,57 @@ release_snapshot() {
   docker manifest push ghcr.io/infracloudio/botkube:${GORELEASER_CURRENT_TAG}
 }
 
+save_pr_image() {
```

Comment:
but why `load`? not sure if I follow.

>  It shouldn't be a big effort, and we could use the save() command for preparing tar archives and attach them to the GH release 🤔

Let's address that on a separate issue/pr related to GitHub release enhancement. As this would require pipelines and changes (e.g. mention how to use the “air gapped” option).

Reply:
Sorry, I meant `save_image` and `load_image`

> Let's address that on a separate issue/pr related to GitHub release enhancement. 

Fine for me 👍